src/cargo/ops/cargo_install.rs

```diff
@@ -461,7 +461,7 @@ pub fn install_list(dst: Option<&str>, config: &Config) -> CargoResult<()> {
     let dst = metadata(config, &dst)?;
     let list = read_crate_list(dst.file())?;
     let mut shell = config.shell();
-    let out = shell.out();
+    let out = shell.err();
```

Comment:
This could in theory break some scripts, and it is also now inconsistent with `cargo --list` which prints to `stdout`. I think that we can do just `println` here, like `cargo --list` does.